dfc/proxy.go

```diff
@@ -849,6 +851,8 @@ func (p *proxyrunner) renamelocalbucket(bucketFrom, bucketTo string, lbmap4ren *
 		glog.Errorf("Failed to store lbmap %s, err: %v", lbpathname, err)
```

Comment:
It is not a question to current changes: should we get back old bucket name to lbmap in case of LocalSave fails. 
1) I remember rollback is implemented on save error in many places of metasync patch but not there
2) it seems, now if save fails, a client receives an error and the client tries again, but error changes to 'bucket does not exist' because lbmap is changed?

Reply:
agree, we need a proper rollback, not just here, may be other places. I will add a comment for now and may be work on some of these rollbacks in a separate change
thanks